packages/turf-boolean-contains/index.js

```diff
@@ -0,0 +1,208 @@
+            return isLineInPoly(geom1, geom2);
+        case 'Polygon':
+            return isPolyInPoly(geom1, geom2);
+        case 'MultiPoint':
+            return isMultiPointInPoly(geom1, geom2);
+        }
+        throw new Error('feature2 ' + geom2.type + ' geometry not supported');
+    default:
+        throw new Error('feature1 ' + geom1.type + ' geometry not supported');
+    }
+};
+
+function isPointInMultiPoint(MultiPoint, Point) {
+    var i;
+    var output = false;
+    for (i = 0; i < MultiPoint.coordinates.length; i++) {
+        if (deepEqual(MultiPoint.coordinates[i], Point.coordinates)) {
+            output = true;
+            break;
+        }
+    }
+    return output;
+}
+
+function isMultiPointInMultiPoint(MultiPoint1, MultiPoint2) {
+    var foundAMatch = 0;
+    for (var i = 0; i < MultiPoint2.coordinates.length; i++) {
+        var anyMatch = false;
+        for (var i2 = 0; i2 < MultiPoint1.coordinates.length; i2++) {
+            if (deepEqual(MultiPoint2.coordinates[i], MultiPoint1.coordinates[i2])) {
+                foundAMatch++;
+                anyMatch = true;
+                break;
+            }
+        }
+        if (!anyMatch) {
+            return false;
+        }
+    }
+    return foundAMatch > 0 && foundAMatch < MultiPoint1.coordinates.length;
+}
+
+// http://stackoverflow.com/a/11908158/1979085
+function isPointOnLine(LineString, Point) {
+    var output = false;
+    for (var i = 0; i < LineString.coordinates.length - 1; i++) {
+        if (isPointOnLineSegment(LineString.coordinates[i], LineString.coordinates[i + 1], Point.coordinates)) {
+            output = true;
+            break;
+        }
+    }
+    return output;
+}
+
+function isMultiPointOnLine(LineString, MultiPoint) {
+    var output = true;
+    for (var i = 0; i < MultiPoint.coordinates.length; i++) {
+        var pointIsOnLine = false;
+        for (var i2 = 0; i2 < LineString.coordinates.length - 1; i2++) {
+            if (isPointOnLineSegment(LineString.coordinates[i2], LineString.coordinates[i2 + 1], MultiPoint.coordinates[i])) {
+                pointIsOnLine = true;
+                break;
+            }
+        }
+        if (!pointIsOnLine) {
+            output = false;
+            break;
+        }
+    }
+    return output;
+}
+
+function isPointInPoly(Polygon, Point) {
+    return inside(Point, Polygon);
+}
+
+function isMultiPointInPoly(Polygon, MultiPoint) {
+    var output = true;
+    for (var i = 0; i < MultiPoint.coordinates.length; i++) {
+        var isInside = isPointInPoly(Polygon, helpers.point(MultiPoint.coordinates[1]));
+        if (!isInside) {
+            output = false;
+            break;
+        }
+    }
+    return output;
+}
+
+// TO DO - Work out how to check if line is in line (can potentially use line overlap module)
+// Also need to make sure lines are exactly the same, eg the second must be smaller than the first
+function isLineOnLine(LineString1, LineString2) {
```

Comment:
```javascript
 t.equal(contains(line1, line3), false, 'A line fails that lies partially outside the other line');
```
[Test fail](https://github.com/Turfjs/turf/pull/707/files#diff-083c7d1fa4cb2170f16604d55e772b48R42) because line should be **completely** on line (returns true because of partial match).

Might be worth adding a `isLineCompletelyOnLine(line1, line2)` method.